src/gui/resources/translations/de.ts

```diff
@@ -0,0 +1,2749 @@
+    <message>
+        <location filename="../../animsettingdialog.ui" line="471"/>
+        <source>Description</source>
+        <translation>Beschreibung</translation>
+    </message>
+    <message>
+        <location filename="../../animsettingdialog.cpp" line="26"/>
+        <source> Animation</source>
+        <translation></translation>
+    </message>
+    <message>
+        <location filename="../../animsettingdialog.cpp" line="33"/>
+        <source>&lt;b&gt;Animation&lt;/b&gt;</source>
+        <translation>&lt;b&gt;Animation&lt;/b&gt;</translation>
+    </message>
+    <message>
+        <location filename="../../animsettingdialog.cpp" line="203"/>
+        <source>&lt;b&gt;Playback&lt;/b&gt;</source>
+        <translation>&lt;b&gt;Wiedergabe&lt;</translation>
+    </message>
+    <message>
+        <location filename="../../animsettingdialog.cpp" line="208"/>
+        <source>Duration:</source>
+        <translation>Dauer:</translation>
+    </message>
+    <message>
+        <location filename="../../animsettingdialog.cpp" line="220"/>
+        <source>Start with mode</source>
+        <translation>Mit Modus starten</translation>
+    </message>
+    <message>
+        <location filename="../../animsettingdialog.cpp" line="228"/>
+        <source>Start with key press</source>
+        <translation>Mit Tastendruck starten</translation>
+    </message>
+    <message>
+        <location filename="../../animsettingdialog.cpp" line="240"/>
+        <source>on pressed key</source>
+        <translation variants="yes">
+            <lengthvariant>Beim Drücken einer Taste</lengthvariant>
+            <lengthvariant>Bei Tastendruck</lengthvariant>
+        </translation>
+    </message>
+    <message>
+        <location filename="../../animsettingdialog.cpp" line="241"/>
+        <source>on whole keyboard</source>
+        <translation>Auf gesamter Tastatur</translation>
+    </message>
+    <message>
+        <location filename="../../animsettingdialog.cpp" line="242"/>
+        <source>on keyboard (once)</source>
+        <translation>Auf Tastatur (einmal)</translation>
+    </message>
+    <message>
+        <location filename="../../animsettingdialog.cpp" line="246"/>
+        <source>every time</source>
+        <translation>jedes Mal</translation>
+    </message>
+    <message>
+        <location filename="../../animsettingdialog.cpp" line="247"/>
+        <source>only once</source>
+        <translation>nur einmal</translation>
+    </message>
+    <message>
+        <location filename="../../animsettingdialog.cpp" line="285"/>
+        <location filename="../../animsettingdialog.cpp" line="297"/>
+        <source>Repeat:</source>
+        <translation>Wiederholen:</translation>
+    </message>
+    <message>
+        <location filename="../../animsettingdialog.cpp" line="295"/>
+        <location filename="../../animsettingdialog.cpp" line="307"/>
+        <source>times</source>
+        <translation>mal</translation>
```

Comment:
This is the dialog we’re talking about:

<img src="https://user-images.githubusercontent.com/30179385/128508484-26937671-da7a-4e65-b648-4119604272db.png" width="250">

> Wiederholen: [ 2 ] mal

I thought of it as in “2-mal wiederholen” or “zweimal wiederholen”.
I think lowercase is correct because it’s not a noun. There’s an even more detailed [article](https://www.duden.de/sprachwissen/sprachratgeber/malMal) in their _Sprachwissen_ section.